ITA/README.md

```diff
@@ -0,0 +1,24 @@
+#  Schrödinger Hat Community
+
+Benvenuti nella community di Schrödinger Hat!
+
+Questo è il punto di partenza per entrare a far parte e contribuire alla community di Schrödinger Hat - migliorando la documentazione, migliorando il codice, tenendo conferenze etc.
+
+## Comunicare
+
+La pagina delle comunicazioni elenca i canali di comunicazione come la chat, i problemi, le conferenze, etc.
+
+## Contribuire
+
+Il primo passo per contribuire è scegliere un progetto. 
+
+Iniziate a unirvi al server Discord e fate due chiacchiere con gli altri utenti.
+
+Ogni repo ha spesso una serie di problemi da risolvere "help wanted" o "good first issue" che possono aiutare i nuovi collaboratori a partecipare.
+
+## Membership
+
+Incoraggiamo tutti i membri a diventare collaboratori ufficiali. Il nostro obiettivo è far crescere una comunità attiva e sana di collaboratori, revisori e proprietari di codice. Per saperne di più sui requisiti e le responsabilità, consultate  la nostra [pagina dedicata ai membri della Community](community-membership.md).
```

Comment:
```suggestion
Incoraggiamo tutti i membri a diventare collaboratori ufficiali. Il nostro obiettivo è far crescere una comunità attiva e sana di contributors, reviewers e code owners. Per saperne di più sui requisiti e le responsabilità, consultate  la nostra [pagina dedicata ai membri della Community](community-membership.md).
```

Essendo parole chiave del mondo opensource io queste 3 parole le lascerei eng @davideimola @TheJoin95

Reply:
Concordo di tenerle in inglese.